docs/syscall_descriptions.md

```diff
@@ -127,6 +127,65 @@ change in descriptions for a particular syscall, the programs that are already i
 the corpus will be kept there, unless you manually clear them out (for example by
 removing the `corpus.db` file).
 
+<div id="tips"/>
+
+## Description tips and FAQ
+
+<div id="names"/>
+
+### Syscall, struct, field, flags names
+
+Stick with existing kernel names for things, don't invent new names if possible.
+
+For example, if there is an existing enum `v4l2_buf_type` in the kernel headers,
+use this name for flags in descriptions as well. The same for structs, unions,
+fields, etc. For syscall variants, use the command name after `$` sign.
+For example, `fcntl$F_GET_RW_HINT`, `ioctl$FIOCLEX`, `setsockopt$SO_TIMESTAMP`.
+
+This makes it easier to look up things in the kernel sources; make things look
```

Comment:
This is an odd way of listing things, and the word "thing" is not great in technical writing. Familiarity and consistency are related.

I would propose to simplify this:
```
Following established naming conventions provides the following benefits: (1) consistency and familiarity of 
names used across kernel interfaces, which also enables searching kernel sources for related names; and (2) 
enable static checking of descriptions (e.g. missed flags or mistyped fields) with [syz-check](/tools/syz-check/check.go).
```

I would move this to the first paragraph, after "Stick with existing kernel names for things, don't invent new names if possible.  [above appended here]".